### DISCUSSION TITLE
讨论区啊

### DISCUSSION THREAD

**Discussion by Qinmei** [Qinmei/qinvideo#106] [闲聊]
测试一下，这个新开的讨论区

**Comment by Tridagger** [comment]
测试！

**Comment by dxhuii** [comment]
这个是根据什么的，才会有讨论区的？

**Reply by Qinmei** [reply]
GitHub内测啊，分阶段开放的吧

**Reply by dxhuii** [reply]
嗯。应该是的，现在应该是按星数，或者项目有没有issues来的吧

**Reply by zhulilia** [reply]
老哥你的Q裙能不能审核一下啊 我安装就开始报错了 年前进群 现在我都还没进去 太难过了

**Reply by zhulilia** [reply]
> 老哥你的Q裙能不能审核一下啊我安装就开始开始报错了年前进群现在我都还没进去太难过了

真的那个教程对于我这种只会一些皮毛的人太难顶了
